### PR TITLE
Throwing paper automatically turns it into a paper plane

### PR DESCRIPTION
**Pull request by MrPerson**
Pickup up a paper plane will automatically unfold it

EDIT: Reasoning:
Standardization of the control scheme.
Simplifying needlessly complex actions.

### CODE REVIEW

**Comment by jughu** [comment]
Powercreep pr :joy:

File: code/modules/paperwork/paperplane.dm [hunk 1]
```diff
@@ -27,14 +27,6 @@
 		internalPaper = new /obj/item/weapon/paper(src)
 	update_icon()
 
-/obj/item/weapon/paperplane/suicide_act(mob/user)
```
Comment (Mervill) [inline code/modules/paperwork/paperplane.dm]: Definitely one of the stranger `suicide_act`'s I've come across. Although it get's creativity points for use of `adjust_eye_damage`.

**Comment by Supermichael777** [comment]
Why though? Paper planes don't magically unfold themselves in real life.

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Mindustry** [comment]
Like @Supermichael777 said, paper planes don't magically fold themselves and unfold themselves.

**Comment by Cyberboss** [comment]
This is actually legitimate powercreep

**Comment by MrPerson** [comment]
What color do you guys want the bike shed to be?

File: code/modules/paperwork/paperplane.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/paperwork/paperplane.dm]: then why are you removing it

**Comment by optimumtact** [comment]
It's not bikeshedding when you're making stupid changes, it's just pointing out the obvious

**Comment by dannno** [comment]
the fuck is bikeshedding

**Comment by optimumtact** [comment]
google it, you spoon

**Comment by dannno** [comment]
fair

**Comment by ExcessiveUseOfCobblestone** [comment]
>please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding

**Comment by Ultimate-Chimera** [comment]
Why not just leave it as alt-click to make it a plane? This doesn't make sense.

**Comment by kilkun** [comment]
i like it.

**Comment by MrPerson** [comment]
You have that backwards, alt-click is the one that makes no sense. Alt-click is the least intuitive, least sensical interaction method available that is to be avoided when it reasonably can be.
And look, a reasonable way to not need it.

Frankly caring about any potential "powercreep" or "muh realisms" or whatever other nonsense is either a joke or caring about the least important thing possible ie bikeshedding.

**Comment by jughu** [comment]
0.2% chance to poke eyes out

**Comment by ExcessiveUseOfCobblestone** [comment]
Why that sounds good and all, you didn't answer the actual question of "why you think these changes would benefit the game?"

**Comment by MrPerson** [comment]
Standardization of the control scheme.
Simplifying needlessly complex actions.

**Comment by optimumtact** [comment]
I don't like it.

**Pull request closed without merging**